code/game/objects/items/devices/spyglasses.dm

```diff
@@ -1,64 +1,305 @@
+Step Three: Press the "Activate Remote View" Button on the side of your SpySpeks™ to open a movable camera display in the corner of your vision, it's just that easy!<br><br><center><b>TROUBLESHOOTING</b><br></center>
+My SpySpeks™ Make a shrill beep while attempting to use!
+A shrill beep coming from your SpySpeks means that they can't connect to any of the ProfitProtektor Mk Ⅱ™, please make sure any of your ProfitProtektor Mk Ⅱ™ is still active, and functional!
+My SpySpeks™ output an error stating they're not connected to a network!
+Simply tap it together with a ProfitProtektor Mk Ⅱ™ of your choosing and their networks will sync up!
+<br>
+<sub><sup>ProfitProtektors Mk Ⅱ™ display vulnerabilities to EMPs and may even be destroyed by them. Nerd Co does not issue refunds for ProfitProtektors Mk Ⅱ™ destroyed in use.</sup></sub>
+	"}
+
+/obj/item/storage/box/rxglasses/spyglasskit/PopulateContents()
+	var/bug_network_id = "[pick(GLOB.phonetic_alphabet)]-[rand(1,999)]"
+	var/obj/item/clothing/glasses/sunglasses/spy/newglasses = new(src)
+	newglasses.bug_network_id = bug_network_id
+	for(var/i in 1 to 5)
+		var/obj/item/video_bug/bug = new(src)
+		bug.bug_network_id = bug_network_id
+	new /obj/item/paper/fluff/nerddocs(src)
 
-/obj/item/clothing/accessory/spy_bug
-	name = "pocket protector"
+/obj/item/video_bug
+	name = "video bug"
 	icon = 'icons/obj/clothing/accessories.dmi'
-	icon_state = "pocketprotector"
-	desc = "An advanced piece of espionage equipment in the shape of a pocket protector. It has a built in 360 degree camera for all your \"admirable\" needs. Microphone not included."
-	var/obj/item/clothing/glasses/sunglasses/spy/linked_glasses
+	icon_state = "camerabug"
+	force = 0
+	throwforce = 0
+	w_class = WEIGHT_CLASS_TINY
+	item_flags = ISWEAPON
+	var/icon_state_clothing = "camerabug_clothing_overlay"
+	var/icon_state_clothing_emissive = "camerabug_clothing_overlay_emissive"
+	desc = "An advanced piece of espionage equipment, which can be attached to most pieces of clothing or hidden under the floor. It has a built in 360 degree camera for all your \"admirable\" needs. Microphone not included."
+	var/x_coord = 0
+	var/y_coord = 0
+	var/obj/item/clothing/attached_to = null
+
+	var/bug_id = null
+	var/bug_network_id = null
+
+	var/being_viewed = FALSE
+	var/obj/item/clothing/glasses/sunglasses/spy/viewing_glasses = null
+
 	var/atom/movable/screen/map_view/cam_screen
 	var/list/cam_plane_masters
 	// Ranges higher than one can be used to see through walls.
 	var/cam_range = 1
 	var/datum/movement_detector/tracker
+	var/atom/movable/screen/background/cam_background
+
+	var/disabled = FALSE
+
+/obj/item/video_bug/Initialize(mapload)
+	update_overlays()
+	var/tile_overlay = image(icon = icon, icon_state = icon_state_clothing, pixel_x = rand(-12, 12), pixel_y = rand(-12, 12))
+	AddElement(/datum/element/undertile, TRAIT_T_RAY_VISIBLE, INVISIBILITY_OBSERVER, tile_overlay, use_anchor = TRUE)
+	bug_id = "[pick(GLOB.posibrain_names)]-[rand(1,999)]"
```

Comment:
Posibrain names? would it be better to use the phonetic alphabet (alpha, beta, charlie, delta, etc.)

Reply:
Phonetic alphabet is already used for network ID names.